addons/osfstorage/views.py

```diff
@@ -58,6 +58,7 @@ def osfstorage_update_metadata(node_addon, payload, **kwargs):
                 "version": "2",
                 "downloads": "1",
                 "checkout": "...",
+                "latestVersionSeen": {"userId": "abc12", "seen": true},
```

Comment:
I'm not sure this belongs. I think this doc is referring to the payload from WB.

Reply:
Yep, the waterbutler payload [changed with the related PR](https://github.com/CenterForOpenScience/waterbutler/blob/develop/waterbutler/providers/osfstorage/metadata.py#L105) to include `latestVersionSeen`

Reply:
Ah ok. Then this is correct. Thanks.